File: Flights.Server/ReadModels/BookingRm.cs
namespace Flights.Server.ReadModels
{
    public record BookingRm
   (
        Guid FlighId,
        string Airline,
        string Price,
        TimePlaceRm Arrival,
        TimePlaceRm Departure,
        int NumberOfBookedSeats,
        string PassengerEmail
   );
}

File: flights.client/src/app/api/models/booking-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from '../models/time-place-rm';
export interface BookingRm {
  airline?: string | null;
  arrival?: TimePlaceRm;
  departure?: TimePlaceRm;
  flighId?: string;
  numberOfBookedSeats?: number;
  passengerEmail?: string | null;
  price?: string | null;
}
